easybuild/tools/module_generator.py

```diff
@@ -289,6 +298,11 @@ def set_alias(self, key, value):
         # quotes are needed, to ensure smooth working of EBDEVEL* modulefiles
         return 'set-alias\t%s\t\t%s\n' % (key, quote_str(value))
 
+    def det_home(self):
+        """
+        Return module-syntax specific code to determine the user's home directory.
+        """
+        return '$env(HOME)'
```

Comment:
maybe this should be generalised to `def get_env(self)`, and then `self.module_generator.get_env('HOME')` should be used?